back/spec/tasks/single_use/migrate_forms_separate_title_description_pages_spec.ignore.rb

```diff
@@ -0,0 +1,229 @@
+require 'rails_helper'
+
+describe 'migrate_custom_forms:separate_title_body_pages rake task' do
+  before { load_rake_tasks_if_not_loaded }
+
+  after { Rake::Task['migrate_custom_forms:separate_title_body_pages'].reenable }
+
+  {
+    %i[title_page title body page] => %i[title_page title body_page body page],
```

Comment:
Good selection of combinations, might be worth adding some extra fields that aren't body or title into the mix

Reply:
I'm not sure if I understand the case you have in mind, which mapping would you add?